spec/factories/orders.rb

```diff
@@ -0,0 +1,27 @@
+FactoryBot.modify do
+  factory :order do
+    trait :with_line_item do
+      transient do
+        variant { FactoryGirl.create(:variant) }
+      end
+
+      after(:create) do |order, evaluator|
+        create(:line_item, order: order, variant: evaluator.variant)
+      end
+    end
+
+    trait :completed do
+      transient do
+        payment_method { create(:payment_method, distributors: [distributor]) }
+      end
+
+      after(:create) do |order, evaluator|
+        order.create_shipment!
+        create(:payment, state: "checkout", order: order, amount: order.total,
+                         payment_method: evaluator.payment_method)
+        order.update_distribution_charge!
+        while !order.completed? do break unless order.next! end
+      end
+    end
+  end
+end
```

Comment:
I think we should namespace these factories and follow spree file names with _factory (I like this simple way to clarify what type of file it is). what do you think?
It would become spec/factories/spree/order_factory.rb
Also because it makes it a bit more clear that this is an extension of https://github.com/openfoodfoundation/spree/blob/step-6a/core/lib/spree/testing_support/factories/order_factory.rb

Also, eventually all order factories here will move to this new file:
https://github.com/openfoodfoundation/openfoodnetwork/blob/master/spec/factories.rb#L294